charts/dgraph/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: dgraph
 version: 0.0.20
```

Comment:
are we going to keep the chart version same ?

Reply:
+1 ^

Reply:
`0.0.20` has never been actually published, so this is not an issue.